### DISCUSSION TITLE
Kailh hot swap positioning

### DISCUSSION THREAD

**Discussion by nethermead** [joshreve/dactyl-keyboard#7] [Ideas]
This is great work!  I just wanted to drop a note that I ran into an issue with the original Clojure forks when I adapted it to use a different style of hot swap socket.  The orientation of the hot swap in the thumb cluster took some iteration to get right, I couldn't just orient it the same way for the two larger thumb keys, the caps wouldn't fit.  Also, prior to rotating them 90 degrees, the very top large thumb socket was too close to the wall and the kailh hot swap piece couldn't fit.  From what I can tell in your images, you might run into a similar issue.

I cut the models at the thumb cluster and just printed that for tests.  I've attached an image of a successful print.
![IMG_1931](https://user-images.githubusercontent.com/56689/124304723-1e695780-db32-11eb-9c2f-0a46dfbb7f72.JPEG)

**Comment by joshreve** [comment]
Interesting thought.  I may want to allow manual rotation of the plate geometry on the cluster.

**Reply by joshreve** [reply]
I did change my orientations a bit, but they may still have issues.  I only messed with the default cluster though.

**Reply by joshreve** [reply]
Added ability to rotate thumb cluster plates in branch cf_thumb.  Will be merged to master shortly and closed.

**Comment by joshreve** [comment]
Added to master and all set if you want to give it a shot.
